tools/mo/openvino/tools/mo/utils/check_ie_bindings.py

```diff
@@ -70,7 +70,10 @@ def import_core_modules(silent: bool, path_to_module: str):
         print("{}: \t{}".format("Model Optimizer version", mo_version))
 
         versions_mismatch = False
-        if mo_version != ie_version:
+        # ie_version has a prefix from CI_BUILD_NUMBER but
+        # if mo_version is from git then it does not have that prefix,
```

Comment:
if you try to upstream changes from master, is already created version.txt file content updated? Per my understanding - not.
Please check this

Reply:
You `version.txt` is not not created when you upstream changes. It's created during build stage on CI and when MO is run on CI infrastructure it takes that `version.txt`.

If there is no `version.txt` (as it's usually for us, developers scenario) then MO tries to get version of itself by running git in a subprocess https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/openvino/tools/mo/utils/version.py#L20-L21

and returned format slightly differs from format from IE.

Reply:
I meant a case when version.txt has been already created after the first upstream and run. When you upstream new changes for the second time version.txt is not updated. Please check.

Reply:
if we a building `model_optimizer`/`MKLDNNPlugin`/`inference_engine`/`ie_api` in our developer environment then version.txt is not used, but instead version is get from `git`.